src/test/java/com/werfen/report/CompareExcelFiles.java

```diff
@@ -0,0 +1,105 @@
+package com.werfen.report;
+
+
+import org.apache.poi.ss.usermodel.*;
+
+import java.util.ArrayList;
+import java.util.Collections;
+import java.util.Iterator;
+import java.util.List;
+
+import static java.util.Objects.nonNull;
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.fail;
+
+public class CompareExcelFiles {
```

Comment:
Just as a pattern for naming this kind of test utilities, I would name this one as ExcelComparator